src/smart-components/group/groups.js

```diff
@@ -144,7 +144,9 @@ const mapDispatchToProps = dispatch => bindActionCreators({
 Groups.propTypes = {
   userIdentity: PropTypes.shape({
     user: PropTypes.shape({
+      /*eslint-disable camelcase*/
```

Comment:
If you want to disable rule for next line, you can go with `eslint-disable-next-line`

Reply:
I will update it in a future pr! thanks!